packages/cli/src/index.ts

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env node
 
-import { Command } from 'commander';
+import { Command, Option } from 'commander';
 import { initCommand, devCommand, DevCommandOptions } from './commands';
 
 const program = new Command();
 
-program.name('novu').description('A CLI tool to interact with the novu API');
+program.name('novu').description('A CLI tool to interact with the Novu Platform');
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** ‏ maybe exchange `Novu Platform` for `Novu Cloud`?

Reply:
I agree.